requirements/app/base.txt

```diff
@@ -1,4 +1,4 @@
-lightning-cloud==0.5.3
+lightning-cloud==0.5.4
```

Comment:
I have [another PR](https://github.com/Lightning-AI/lightning/pull/14697) that bumps this to 0.5.5. Before merge, please verify whether that PR is merged or not, to avoid overwriting it and downgrading to 0.5.4